code/src/Pizza.ts

```diff
@@ -0,0 +1,14 @@
+import { Consumable } from './Consumable';
+
+export class Pizza extends Consumable {
+  numberOfSlices: number;
+  slicesEaten: number = 0;
+  constructor(numberOfSlices: number, spoiled: boolean) {
+    super('Pizza', 0, 0, spoiled);
+    this.numberOfSlices = numberOfSlices;
+  }
+
+  eat(): string {
+    return 'eat';
```

Comment:
This method should use variables "numberOfSlices" and "slicesEaten".

Reply:
fixed